packs/spell-effects/effect-albatross-curse-circumstance-bonus.json

```
@@ -0,0 +1,44 @@
+{
+    "_id": "QFeJq767fymcw6LA",
+    "img": "icons/creatures/birds/dove-pigeon-flying-white.webp",
+    "name": "Effect: Albatross Curse Circumstance Bonus",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p><span style=\"font-family:Roboto, sans-serif\">You and allies within 30 feet of the target gain a +1 circumstance bonus to attacks against the target.</span></p>"
```

```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.spells-srd.Item.Albatross Curse].</p>\n<p>You and allies within 30 feet of the target gain a +1 circumstance bonus to attacks against the target.</p>"
```

Edit in HTML mode or paste with control-shift-v to avoid those